lib/Try/Tiny.pm

```diff
@@ -29,6 +29,8 @@ BEGIN {
   *_HAS_SUBNAME = ($su || $sn) ? sub(){1} : sub(){0};
 }
 
+our @_finally_guards;
```

Comment:
I don't think that's going to be a problem in practice. Try::Tiny already contains the following fondleables: `$VERSION`, `@EXPORT`, `_subname`, `_HAS_SUBNAME`, and an entire hidden package: `Try::Tiny::ScopeGuard`.